### DISCUSSION TITLE
OGC API - Features. Initial integration published in production instance - Updated

### DISCUSSION THREAD

**Discussion by jenriquesoriano** [INSPIRE-MIF/helpdesk-validator#767] [Announcements]
This message is to announce the deployment of the initial OGC API-Features validation in the [staging instance](http://staging-inspire-validator.eu-west-1.elasticbeanstalk.com/etf-webapp/home/index.html) instance .

This development covers the integration of the cases produced and published in the OGC Team Engine through the use of the OGC API - Features ETSs via the REST interface of the TEAM engine API. This integration takes as an input a landing page for an API, sends it to the TEAM Engine deployment maintained by the OGC, and collects the results transforming them into a more easily readable TestReport.

Please note that the INSPIRE reference validator is acting as an interface to the TEAM Engine API, and these tests are processed on that remote deployment. Due to this, the availability of the service and the system load is out of control of the validator. Tests can take up to several minutes and timeouts are possible.

**Comment by jenriquesoriano** [comment]
Please note that the initial integration of OGC API - Features has been deployed in the Production instance as per v2022.0

**Comment by jsaligoe** [comment]
Hello @jenriquesoriano, Perhaps I'm missing something on how to run the test. Can you provide example implementations passing the INSPIRE Reference Validator?  We have tried a few live test data that have passed OGC compliance but I am unable to generate a passing result (e.g., interactive instruments and lat/long  https://www.ogc.org/resource/products/compliant?display_opt=1&specid=1022 ). thanks

**Comment by jenriquesoriano** [comment]
Dear @jsaligoe  , 

thanks for your contribution. 
We have tested some OGC API - Features using the landing page url and the results seem to be consistent.
Could you provide us a concrete landing page which is producing the issue you comment?
We would also reccomend to create a separate issue at this regard, as this is a "for-information" issue.

Best regards,
